template/build/webpack.prod.conf.js

```diff
@@ -41,6 +42,22 @@ var webpackConfig = merge(baseWebpackConfig, {
     new ExtractTextPlugin({
       filename: utils.assetsPath('css/[name].[contenthash].css')
     }),
+	new PurifyCSSPlugin({
```

Comment:
It's probably worth adding a comment above here to let the developer's know what this plugin is doing.